### PR TITLE
Add 8.3 What's New for Observability

### PR DESCRIPTION
**Pull request by ollyhowell**

### CODE REVIEW

File: docs/en/observability/whats-new.asciidoc [hunk 1]
```diff
@@ -5,13 +5,28 @@ Here are the highlights of what's new and improved in {minor-version}.
...
+{observability-guide-all}/7.17/whats-new.html[7.17] |
 
 // tag::whats-new[]
 
 // What's new content goes in here. Don't uncomment or remove the tags surrounding this content :)
 
+[discrete]
+== APM Server introduces support for OpenTelemetry protocol over HTTP transport
+Starting in version 8.3, APM Server supports the OpenTelemetry protocol over
+HTTP transport ([OTLP/HTTP](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#otlphttp).
```
Comment (bmorelli25) [inline docs/en/observability/whats-new.asciidoc:21]:
```suggestion
HTTP transport https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#otlphttp[OTLP/HTTP].
```

File: docs/en/observability/whats-new.asciidoc [hunk 2]
```diff
@@ -5,13 +5,28 @@ Here are the highlights of what's new and improved in {minor-version}.
...
+This new support is in addition to the existing OTLP
+protocol over gRPC transport ([OTLP/gRPC](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#otlpgrpc).
+Both OTLP/gRPC and OTLP/HTTP use the
+same ProtoBuf payload.
+
+The OTLP/HTTP protocol is useful for users with gRPC-based infrastructure
+limitations, or users who want to take advantage of the smaller HTTP dependency.
+These benefits and additional details are outlined in the official [OpenTelemetry
+HTTP proposal](https://github.com/open-telemetry/oteps/blob/main/text/0099-otlp-http.md#motivation).
```
Comment (bmorelli25) [inline docs/en/observability/whats-new.asciidoc:30]:
```suggestion
These benefits and additional details are outlined in the official https://github.com/open-telemetry/oteps/blob/main/text/0099-otlp-http.md#motivation[OpenTelemetry
HTTP proposal].
```

File: docs/en/observability/whats-new.asciidoc [hunk 3]
```diff
@@ -5,13 +5,28 @@ Here are the highlights of what's new and improved in {minor-version}.
 
 coming[8.0.0]
```
Comment (bmorelli25) [inline docs/en/observability/whats-new.asciidoc:6]:
```suggestion
```

**Review by colleenmcginnis — APPROVED** [review]
Looks good! I think you can remove the backport label now that this PR is targeting `elastic:8.3`.

**Comment by ollyhowell** [comment]
Thanks for your help @colleenmcginnis !

**Pull request merged**